installation/rasppi.md

```diff
@@ -6,5 +6,78 @@ layout: documentation
 
 # Installation on a Raspberry Pi
 
-to be done...
+There’s no pre-configured image for openHAB, so installation is done the old fashioned way via a command line.
+
+Start with the latest (full) Raspbian SD image (not the “lite” version, these don’t include the Java Virtual Machine). Get your network cable plugged in, then boot up, and navigate through SSH. Run:
+
+`sudo raspi-config`
+
+Expand the filesystem; and from the advanced menu, change the memory split to 16. When are done, restart, and as good practice, run a full update
+
+Install openHAB on Linux, see [Installation on Linux through APT](http://docs.openhab.org/installation/apt.html)
+
+Curiously, everything was installed as owned by “root”. We need to fix that with the following commands.
```

Comment:
@theoweiss Is this correct? I thought that APT uses the openhab user for the installation?